validationtools/validationtools/loadBank/_loadBank.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import pyvisa as visa
+import time
+import numpy as np
+from tqdm import tqdm
+import pandas as pd
+
+class LoadBank:
+    def __init__(self):
+        '''
+        Represents a Rigol DL3021 programmable DC electronic load.
+        '''
+        resources = visa.ResourceManager()
+
+        resourceTag = ''
+        for x in resources.list_resources():
+            print(x)
```

Comment:
this print should be probably left to only debug/verbose mode. Make use of `logging` python library for that.

Reply:
done!